sphinx_gallery/gen_rst.py

```diff
@@ -13,18 +13,19 @@
 
 """
 from __future__ import division, print_function, absolute_import
+from . import glr_path_static
+from .backreferences import write_backreferences, _thumbnail_div
+from textwrap import dedent
 from time import time
 import ast
+import hashlib
 import os
 import re
 import shutil
-import traceback
-import sys
 import subprocess
+import sys
+import traceback
 import warnings
-from textwrap import dedent
-from . import glr_path_static
-from .backreferences import write_backreferences, _thumbnail_div
```

Comment:
> On 1 janv. 2016, at 22:25, Óscar Nájera <notifications@github.com> wrote:
> 
> In sphinx_gallery/gen_rst.py:
> 
> >  import warnings
> > -from textwrap import dedent
> > -from . import glr_path_static
> > -from .backreferences import write_backreferences, _thumbnail_div
> Because I did an alphabetical sort. I can chage it, no string feeling for this.
> 
> Given your ordering logic, should the imports of matplotlib, basestring and textwrap that have more complicated import instructions be also before the local files, or after all that import list?
> 

Matplotlib is not a local file it should be Just above locals and after standard library imports 
> —
> Reply to this email directly or view it on GitHub.
>